builtin/logical/pki/test_helpers.go

```diff
@@ -181,42 +180,6 @@ func getParsedCrl(t *testing.T, client *api.Client, mountPoint string) *pkix.Cer
 	return getParsedCrlAtPath(t, client, path)
 }
 
-func getParsedCrlForIssuer(t *testing.T, client *api.Client, mountPoint string, issuer string) *pkix.CertificateList {
```

Comment:
I had liked the idea of this function, we should really turn the existing one into an optional parent cert and pass it in when we have it.